lib/facets/aggs-documents.js

```diff
@@ -15,18 +15,6 @@ module.exports = function (queryParams) {
             terms: { field: 'fonds.summary_title' }
           }
         }
-      },
```

Comment:
ok sounds good, if the index changes we will add the organisation facets if it's still necessary, but for now let's delete the unused code.